boards/arm/disco_l475_iot1/disco_l475_iot1_defconfig

```diff
@@ -54,8 +55,15 @@ CONFIG_BT_SPI_CHIP_SELECT_DEV_NAME="GPIOD"
 CONFIG_BT_SPI_CHIP_SELECT_PIN=13
 CONFIG_BT_SPI_RESET_DEV_NAME="GPIOA"
 CONFIG_BT_SPI_RESET_PIN=8
+CONFIG_BT_BLUENRG_ACI=y
 
+# Disable Flow control
+CONFIG_BT_HCI_ACL_FLOW_CONTROL=n
+CONFIG_BT_HCI_VS_EXT=n
 
 # sensors configuration
 CONFIG_LIS3MDL_TRIGGER_NONE=y
 CONFIG_HTS221_TRIGGER_NONE=y
+
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_BT_SPI_MAX_CLK_FREQ=2000000
```

Comment:
Move this one up with CONFIG_BT_SPI_xxx friends

Reply:
DONE, available with new update